### DISCUSSION TITLE
How to send a request based on the output of another request ?

### DISCUSSION THREAD

**Discussion by omarhashem123** [projectdiscovery/nuclei-templates#6376] [Q&A] [answered]
I'm just trying to get a value from the first request and put it in the second request and then match a specific value in the second request to find out the vulnerability
for example
First request to: http://127.0.0.1/test.php
In response to this request, I want to extract a token with regex and then
Second request: http://127.0.0.1/vuln.php?param=token-extracted-from-the-first-request
Then matching on the response of the second request with a regex to detect if the vulnerability exists

**Comment by olearycrew** [comment, marked as answer]
See workflows and variables: https://nuclei.projectdiscovery.io/templating-guide/variables/ & https://nuclei.projectdiscovery.io/templating-guide/workflows/
